mvn-resolver-transport-http3/src/main/java/com/artipie/aether/transport/http3/HttpTransporter.java

```diff
@@ -301,13 +297,19 @@ private ContentResponse makeRequest(HttpMethod method, TransportTask task)
                 new BasicAuthentication.BasicResult(this.baseUri, this.authInfo[0], this.authInfo[1])
             );
         }
-        final ContentResponse response = this.client.newRequest(url).method(method).headers(httpFields -> {
-            System.err.printf("\tCustom HEADER HttpTransporter.makeRequest() called! fields: %d; URL: %s%n", httpFields.size(), url);
-            final Object token = this.state.getUserToken();
-            if (token != null) {
-                httpFields.add(LocalState.USER_TOKEN, token.toString());
-            }
-        }).send();
+        final Request request = this.client.newRequest(url);
+        final ContentResponse response;
+        try {
+            response = request.method(method).headers(httpFields -> {
+                System.err.printf("\tCustom HEADER HttpTransporter.makeRequest() called! fields: %d; URL: %s%n", httpFields.size(), url);
+                final Object token = this.state.getUserToken();
+                if (token != null) {
+                    httpFields.add(LocalState.USER_TOKEN, token.toString());
+                }
+            }).send();
```

Comment:
hm, if method is `PUT` maven `TransportTask` should hold some data (checksums or jar) to send, I do not see, were this data is send?